staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_progress_test.go

```diff
@@ -48,17 +48,29 @@ func TestConditionalProgressRequester(t *testing.T) {
 	var requestsSent int32
 
 	logger.Info("No progress requests if no-one is waiting")
+	for !clock.HasWaiters() {
```

Comment:
I'm not sure I'm following this - why the clock should have waiter? Can you add a comment explaining this?

Reply:
Means Ticker was created. We want to wait for ticker to be created before we start progress the clock.